.travis.yml

```diff
@@ -1,7 +1,8 @@
 sudo: false
 language: ruby
 before_install:
-  - gem update bundler
+  -  gem uninstall bundler
+  -  gem install bundler -v '1.17.3'
```

Comment:
Please can you move this in a dedicated Pull Request?
(cf "Pull Requests - Code of Conduct": https://forest.slite.com/app/channels/-KdBU14ToWBAerCk_HGS/notes/QFFoNQ1pOg#68229e06)

Reply:
Done ForestAdmin/forestadmin#342